GateMaterials.db

```diff
@@ -292,10 +471,19 @@ Adipose: d=0.92 g/cm3 ; n=11
         +el: name=Calcium    ; f=0.001
         +el: name=Scandium   ; f=0.0
         +el: name=Titanium   ; f=0.0
-        +el: name=Vandium    ; f=0.0
+        +el: name=Vanadium   ; f=0.0
         +el: name=Chromium   ; f=0.0
         +el: name=Manganese  ; f=0.0
 
+Adipose_Tissue: d=1.95 g/cm3 ; n=7
```

Comment:
Prefer G4_ADIPOSE_TISSUE_ICRP?

Reply:
removing both `Adipose` and `Adipose_Tissue` in favor of `G4_ADIPOSE_TISSUE_ICRP`